torch/csrc/jit/pickler.cpp

```diff
@@ -736,19 +736,19 @@ OpCode Unpickler::readInstruction() {
       switch (class_name) {
         case PicklerClass::TENSOR:
           stack_.emplace_back(
-              tensor_table_->at(data->elements().at(0).toInt()));
+              tensor_table_->at(data->elements().get(0).toInt()));
```

Comment:
why is this API arbitrarily different?

Reply:
`ListPtr` intentionally does not have the `std::vector` API to try and avoid people thinking it has value behavior instead of reference behavior.